src/pages/permissions/panels/EntitlementOptionLabel.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from "react";
+
+interface Props {
+  entitlement: string;
+  description: string;
+}
+
+const EntitlementOptionLabel: FC<Props> = ({ entitlement, description }) => {
+  return (
+    <div className="label">
+      <span title={entitlement} className="entitlement u-truncate">
+        {entitlement}
+      </span>
+      <span
+        title={description}
+        className="entitlement-description u-truncate u-text--muted"
```

Comment:
The muted doesn't work in case the row is selected. The contrast of text to background seems too low. There is probably an inverted class or something that can be set for muted text to be handling this case well.